README.md

```diff
@@ -15,6 +15,10 @@ The MSAL library for Android gives your app the ability to use the [Microsoft Cl
 
 > Looking for developers interested in providing early feedback on a x-platform implementation of MSAL written in C++ and Java, callable from Java, Kotlin and C++.  If you're interested please please contact shoatman@microsoft.com.
 
+***06/25/2021***
+- Silent requests inadvertently serialized in MSAL v2.0.8-v2.0.12, Common v3.1.2-v3.4.3. This will be fixed in an upcoming release, tentatively scheduled for next week.
```

Comment:
Missed word: Silent requests were inadvertently serialized....

I think we need to add a strong warning.  Do not use these versions and provide current recommended version... which presumably would be 2.0.7

Reply:
Added a suggestion below.